draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -140,6 +140,8 @@ For any host ICE candidate gathered by a browsing context as part of {{RFC8445}}
 
 7. Replace the IP address of the ICE host candidate with the name with ".local" appended to it. Expose the candidate.
 
+ICE host candidates generated using this procedure MUST have an unique mDNS name generated for each host candidate IP address, specifically an ICE agent using an interface with both IPv4 and IPv6 addresses MUST expose a different mDNS name for each address.
```

Comment:
s/an unique/a unique/?

Reply:
Also the wording could be improved to something like:
This procedure MUST ensure that a mMDNS name should be used to replace only one IP address. 
Specifically an ICE agent using an interface with both IPv4 and IPv6 addresses MUST expose a different mDNS name for each address.